### ISSUE TITLE
detect result

### ISSUE THREAD

**Issue by zhanglaplace** [toandaominh1997/EfficientDet.Pytorch#38]
您好，这边测试了一个weights/checkpoint_VOC_efficientdet-d1_37.pth的模型，结果是这个样子的，有几个好像未检测到
![image](https://user-images.githubusercontent.com/12178974/70979058-e9cab200-20eb-11ea-8728-ee846b337360.png)

**Comment by toandaominh1997** [comment]
hi @zhanglaplace,
I just trained on VOC at 37 epoch, it really hasn't optimized yet,
If you get better for detection, you can try this weights: [weights at epoch 65](https://drive.google.com/open?id=1hWqQf99K49FCeMJ-WSFVR_LFr7VfYCuE)
And this is results for this epoch
![demo](https://user-images.githubusercontent.com/18400648/70982905-6ceb0880-20ea-11ea-9a99-6a4fbec4f49a.png)
You can try it.:v
Anyway, I will public a better weight when complete training.

**Comment by zhanglaplace** [comment]
@toandaominh1997  
Looking forward to your new model.

**Comment by Dewey1994** [comment]
hello!
can you provide your train loss figures if you complete the training?
i have training 130 epoch, but it hasn't optimized yet
thank you!

**Comment by xafarranxera** [comment]
Good morning,

I couldn't run the demo with the weights for D1 at 65 epoch, as checkpoint hasn't any "parser" inside

Has anybody got this solved?

> hi @zhanglaplace,
> I just trained on VOC at 37 epoch, it really hasn't optimized yet,
> If you get better for detection, you can try this weights: [weights at epoch 65](https://drive.google.com/open?id=1hWqQf99K49FCeMJ-WSFVR_LFr7VfYCuE)
> And this is results for this epoch
> ![demo](https://user-images.githubusercontent.com/18400648/70982905-6ceb0880-20ea-11ea-9a99-6a4fbec4f49a.png)
> You can try it.:v
> Anyway, I will public a better weight when complete training.

**Comment by inFreedom92** [comment]
@xafarranxera 
The code has been changed since the author replied, so you should retrain it your self.
I think...

**Comment by xafarranxera** [comment]
Yes... I changed the code a bit to solve this, but I got a permission error for downloading the necessary efficientnet-d1 model. We're gonna wait for the d1 trained version from the author, luckily.

**Comment by inFreedom92** [comment]
@xafarranxera 
You could try this.
#110

**Comment by xafarranxera** [comment]
@inFreedom92 Working now. Thank you so much.

**Comment by xafarranxera** [comment]
@inFreedom92 Regarding that, have you been able to get any good pretrained model for D1-D7 versions?

**Comment by inFreedom92** [comment]
@xafarranxera 
I'm currently training with my own dataset.
I'm trying to detect cars and license plates.
The model that inference below image is by using this repository's code about a month ago.
But the author of this repository changed some code so I'm trying to catch up and make my model better.
In my opinion, it is not good enough.
![image](https://user-images.githubusercontent.com/13886707/73259188-8c597300-420a-11ea-95b8-cf5b31d7f381.png)
